docs/PUBLISHING.md

```diff
@@ -27,6 +27,8 @@ We follow a [git-flow](https://nvie.com/posts/a-successful-git-branching-model/)
 
 #### Cutting a release branch
 
+_Warning: In step 4 you will see a message in your terminal with instructions on publising the package. Ignore the message in your terminal and follow the instructions in this guide instead. We send the release branch through the pull request review process before publishing._
```

Comment:
(Not related to this change)

Just realized standard-version is deprecated 😬. The repo recommends https://github.com/googleapis/release-please

Reply:
Created ticket https://app.shortcut.com/czi-edu/story/225066/move-eds-from-standard-release-to-release-please-or-maintained-alternative

Reply:
Thanks!